### PR TITLE
SUSE SMT Documentation

### PR DESCRIPTION
**Pull request by parthaa**
This commit adds docs to mirror content from SUSE SCC via the SMT tool.

### CODE REVIEW

**Comment by johnpmitsch** [comment]
formatting looks off

f

**Comment by parthaa** [comment]
@johnpmitsch updated

**Comment by sean797** [comment]
Do we need to have anything for registering and updating client SUSE systems?

File: _includes/plugins/katello/sidebar_nightly.html [hunk 1]
```diff
@@ -142,6 +142,11 @@
         5.17 Red Hat Content
       </a>
     </li>
+    <li class="item">
+      <a href="/plugins/katello/nightly/user_guide/suse_content/index.html">
+        5.17 SUSE Content
```
Comment (tbrisker) [inline _includes/plugins/katello/sidebar_nightly.html]: Shouldn't this be 5.18?

**Comment by bbuckingham** [comment]
@sean797, we have a separate task to cover clients.  That should be coming soon as well.  :)

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  # smt-mirror
+```
+- This should install an apache2 on that host
+```
+  # service apache2 restart
+```
+You should be able to navigate to the mirror by browsing to `http://<fqdn>/repo/`
+
+## Create Products and Repositories
+- Follow the steps listed [here](/plugins/katello/{{ page.version }}/user_guide/content/index.html) to Create Products and repositories
```
Comment (johnpmitsch) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: This goes to the same link as below, I don't think we really need this bullet point and can just keep the second one

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- Follow the steps listed [here](/plugins/katello/{{ page.version }}/user_guide/content/index.html) to Create a yum repository with the repository url pointing to the appropriate location. For example `http://<fqdn>/repo/SUSE/Products/SLE-SERVER/12-SP3/x86_64/product/`
```
Comment (johnpmitsch) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: s/Create/create/

**Review by johnpmitsch — CHANGES_REQUESTED** [review]
Some small changes, but otherwise looks good!

**Comment by parthaa** [comment]
@johnpmitsch updated

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  # smt-mirror
+```
+- This should install an apache2 on that host
+```
+  # service apache2 restart
+```
+You should be able to navigate to the mirror by browsing to `http://<fqdn>/repo/`
+
+## Create Products and Repositories
+- Follow the steps listed [here](/plugins/katello/{{ page.version }}/user_guide/content/index.html) to Create Products and Repositories
```
Comment (johnpmitsch) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: this still needs s/Create/create

**Review by johnpmitsch — APPROVED** [review]

**Comment by johnpmitsch** [comment]
thanks @parthaa!

**Pull request merged**